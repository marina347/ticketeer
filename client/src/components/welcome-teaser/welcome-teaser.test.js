import React from "react";
import { shallow } from "enzyme";
import WelcomeTeaser from "./welcome-teaser.component";

it("Should render WelcomeTeaser component", () => {
  expect(shallow(<WelcomeTeaser displayName="name" />)).toMatchSnapshot();
});
