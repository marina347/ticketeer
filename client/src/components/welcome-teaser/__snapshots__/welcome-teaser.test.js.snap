// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeTeaser component Should render WelcomeTeaser component 1`] = `
<div
  className="welcome-teaser"
  id="welcome_teaser"
>
  <SecondaryHeading
    text="Hello John!"
  />
  ;
</div>
`;
